modules/nf-core/gsea/gsea/main.nf

```diff
@@ -0,0 +1,77 @@
+process GSEA_GSEA {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::gsea=4.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gsea:4.3.2--hdfd78af_0':
+        'quay.io/biocontainers/gsea:4.3.2--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(gct), path(cls), path(gene_sets)
+
+    output:
+    tuple val(meta), path("*/*.rpt")                                                                                                        , emit: param_log
+    tuple val(meta), path("*/index.html")                                                                                                   , emit: index
+    tuple val(meta), path("*/heat_map_corr_plot.html")                                                                                      , emit: heat_map_corr_plot
+    tuple val(meta), path("*/gsea_report_for_reference_*.tsv"),  path("*/gsea_report_for_target_*.tsv")                                     , emit: report_tsvs
+    tuple val(meta), path("*/gsea_report_for_reference_*.html"),  path("*/gsea_report_for_target_*.html")                                   , emit: report_htmls
```

Comment:
These should be separate outputs channels, you can combine them after [^1]

[^1]: https://nf-co.re/docs/contributing/modules#inputoutput-options point 5